src/dealDashboard/discussionsService.ts

```diff
@@ -408,7 +408,10 @@ export class DiscussionsService {
         localStorage.getItem("discussionToken"),
         commentId,
       );
-      this.comments = this.comments.filter(comment => comment._id !== commentId);
+      this.comments = this.comments.filter((comment: IComment) => comment._id !== commentId);
+      this.comments.map((comment: IComment) => {
```

Comment:
nitpick, optional

```suggestion
      this.comments.forEach((comment: IComment) => {
```

Because not returning anything

Reply:
agreed